scss/calorie-counter.scss

```diff
@@ -0,0 +1,86 @@
+@import './mixins';
```

Comment:
Why do you have two files for this?
This should be a single file that manages all the styles for mobile, tablet and desktop for your component using a mobile first approach. Please adjust,